autopengui.py

```diff
@@ -51,16 +57,106 @@
 sdr = ('gnu radio', 'gqrx')
 mis = ('katoolin')
 all_tools = can + (b_w,) + (sdr,) + (mis,)
-
 tools_to_install = []
 broken = []
+chosen = ''
+cars = []
+connection = 1
+def set_chosen(input):
+	global chosen
+	chosen = input
+
+def get_chosen():
+	return chosen
+
+try:
+    db = MySQLdb.connect(host="autopen.c5eqmploekdx.us-east-2.rds.amazonaws.com", user="SHIFT", passwd="%&12SHilaE7", db="autopen", port=3306)
```

Comment:
I don't think that including the username and password of our server is a good idea here. This would give anyone access to the database.